qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml

```diff
@@ -59,14 +59,14 @@ tasks:
       - ceph orch ps
       - ceph orch ls
       - ceph versions
-      # to make sure mgr daemons upgrade is fully completed, including being deployed by a mgr on new new version
+      # to make sure mgr daemons upgrade is fully completed, including being deployed by a mgr on new version
       # also serves as an early failure if manually upgrading the mgrs failed as --daemon-types won't be recognized
       - ceph orch upgrade start --image quay.ceph.io/ceph-ci/ceph:$sha1 --daemon-types mgr
       - while ceph orch upgrade status | jq '.in_progress' | grep true && ! ceph orch upgrade status | jq '.message' | grep Error ; do ceph orch ps ; ceph versions ; ceph orch upgrade status ; sleep 30 ; done
       # verify only one version found for mgrs and that their version hash matches what we are upgrading to
       - ceph versions | jq -e '.mgr | length == 1'
       - ceph versions | jq -e '.mgr | keys' | grep $sha1
-      # verify overall we still se two versions, basically to make sure --daemon-types wans't ignored and all daemons upgraded
+      # verify overall we still se two versions, basically to make sure --daemon-types wasn't ignored and all daemons upgraded
```

Comment:
There was a request to expand contractions. I only did it in one case. I can do it more generally, but given how large the main PR is, and that even when split, the PRs are still sizable, my preference is to not add too many changes at once. I can always return (especially once people are more comfortable w/ me making large changes).